libs/wingsdk/src/shared-aws/permissions.ts

```diff
@@ -95,7 +95,10 @@ export function calculateBucketPermissions(
     ops.includes(cloud.BucketInflightMethods.EXISTS) ||
     ops.includes(cloud.BucketInflightMethods.TRY_GET) ||
     ops.includes(cloud.BucketInflightMethods.TRY_GET_JSON) ||
-    ops.includes(cloud.BucketInflightMethods.TRY_DELETE)
+    ops.includes(cloud.BucketInflightMethods.TRY_DELETE) ||
+    // get requires list permissions too https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/s3/command/GetObjectCommand/
```

Comment:
According to the docs you linked, having List permissions only changes the error you get  when an object isn't found. So it's not actually needed. It's possible I'm misreading it though.
 
Did you config that this change fixes the original issue?

Reply:
From the original issue:

> I expected this:
> No error and correct permissions

Not just a different (better) error